webapp/portlet/src/main/webapp/vue-apps/common/components/ActivityShareDrawer.vue

```diff
@@ -132,6 +132,9 @@ export default {
             displayName: space && space.profile && space.profile.fullName,
             avatarUrl: space && space.profile && space.profile.avatarUrl,
           })), this.currentApp);
+          if (localStorage.getItem('activity-message')) {
```

Comment:
It would be better to add in `exo-activity-rich-editor` an attribute for context name (`contextName`: `activityComposer`, `kudos`, `activityComment`, `activityShare` ....) and a boolean props to tell if we need to reuse the draft value or not (`useDraftManagement`). and we weill need to rename the localStorage attribute name to be: `activity-message-${this.contextName}`. By doing this we will make sure to preserve a Draft per context.